### PR TITLE
Misc UI

### PR DESCRIPTION
**Pull request by ghost**
1) Navbar added to Sign page and Register page.
2) Styling added for support page same as of transfer page.

### CODE REVIEW

File: src/main/react-front-end/src/views/Support/SupportComponent.js [hunk 1]
```diff
@@ -168,7 +178,10 @@ export default class SupportComponent extends Component{
           </div>
 
         </ValidatorForm>
-      </Card>
+        {/*</Panel.Body>*/}
+      </Panel>
+          </Col>d
```
Comment (LinusCastelino) [inline src/main/react-front-end/src/views/Support/SupportComponent.js]: What is this 'd' for?

**Comment by LinusCastelino** [comment]
@SaiAindla Can you point out where the Navbar is added to the sign in page? I dont find any related changes in this PR

File: src/main/react-front-end/src/MainComponent.js [hunk 2]
```diff
@@ -48,6 +50,7 @@ export default class MainComponent extends Component {
 
         <div style={{ marginTop: '50px', display: 'block' }}>
           <Switch>
+            <Route path='/account' component={AccountControlComponent}/>
```
Comment (ghost) [inline src/main/react-front-end/src/MainComponent.js:53]: Removing the Route path of AccountControlComponent from App.js and adding it in the MainComponent.js added Navbar to Sign page and register page.

File: src/main/react-front-end/src/views/Support/SupportComponent.js (hunk 1 above)
Comment (ghost) [reply src/main/react-front-end/src/views/Support/SupportComponent.js]: Sorry, that was a typo mistake from me.
Do you want me to raise the PR again?

File: src/main/react-front-end/src/MainComponent.js (hunk 2 above)
Comment (LinusCastelino) [reply src/main/react-front-end/src/MainComponent.js:53]: Understood now

File: src/main/react-front-end/src/views/Support/SupportComponent.js (hunk 1 above)
Comment (LinusCastelino) [reply src/main/react-front-end/src/views/Support/SupportComponent.js]: Please correct and recommit.

Comment (ghost) [reply src/main/react-front-end/src/views/Support/SupportComponent.js]: Okay

**Review by LinusCastelino — APPROVED** [review]
Looks good!

**Comment by LinusCastelino** [comment]
[Misc UI changes](https://trello.com/c/TjxgXvWj/42-misc-ui-changes)

**Comment by msaianand** [comment]
[Front end validation for reset password](https://trello.com/c/uOetaujz/56-front-end-validation-for-reset-password)

**Pull request merged**